src/components/TextInput/TextInput.js

```diff
@@ -163,7 +163,7 @@ const AddonRegularText = styled(BaseText)`
 const AddonIcon = styled(Icon)`
   font-size: 16px;
   color: ${({ theme }) => theme.colors.basic010};
-  margin-right: 9px;
+  padding-right: 9px;
```

Comment:
Is that change necessary? It does not seem to cause any visual effect?

Reply:
This was regarding 5 point.